content/source/docs/enterprise/api/run.html.md

```diff
@@ -533,6 +533,51 @@ curl \
   https://app.terraform.io/api/v2/runs/run-DQGdmrWMX8z9yWQB/actions/force-cancel
 ```
 
+## Forcefully execute a run
+
+`POST /runs/:run_id/actions/force-execute`
+
+Parameter | Description
+----------|------------
+`run_id`  | The run ID to execute
+
+The `force-execute` action is analogous to executing the latest queued run after discarding or canceling all prior runs that are not already complete. It initiates the same actions as the "Run this plan now" button at the top of the view of a pending run.
+
+This endpoint enforces the following prerequisites:
+
+* the target run is in the Pending state
+* the workspace is locked by another run
+* the run locking the workspace can be discarded 
+
+Runs can currently be discarded if they are in the Planned, Policy Checked, or Policy Override states.
```

Comment:
Should we remove `currently`? I think that's the assumption when reading API docs. Saying "currently" explicitly makes me feel like this is about to change.

Reply:
I went ahead and removed this - I definitely see your concern in keeping it updated. I do have some reservations that not explaining it leaves us open for the question "what is a discardable state?" But we can always add something about it later, if the question is frequent.